### DISCUSSION TITLE
standard os mouse cursor

### DISCUSSION THREAD

**Discussion by N3C2L** [madprops/grasshopper#72] [Ideas]
In other tab sidebar extensions of firefox, the normal OS mouse cursor is used if hovered over a tab:
https://github.com/piroor/treestyletab

In Grasshopper the cursor changed when hovered over a tab. It's the same cursor that appears when hovered over an image in a website content of firefox. However this cursor is not that accurate to navigate.

I would appreciate an option to change that.

**Comment by madprops** [comment]
Ok I guess that's cleaner. I'll just change that, no setting.

**Reply by N3C2L** [reply]
Wonderful, thanks.

**Comment by N3C2L** [comment]
This idea has been implemented, so I can close this thread.
